examples/plain/scripts/upload-debug-android.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+API_KEY=YOUR_API_KEY_HERE # set your own API key here
+APP_VERSION=1 # set your own app version here (using the Android versionCode)
+
+# Download debug source maps from Metro bundler
+curl "http://localhost:8081/index.bundle?platform=android&dev=true&minify=false" > android-debug.bundle
+curl "http://localhost:8081/index.bundle.map?platform=android&dev=true&minify=false" > android-debug.bundle.map
+
+# Upload source maps to Bugsnag, making sure to specify the correct app-version.
+bugsnag-sourcemaps upload \
```

Comment:
`bugsnag-sourcemaps` needs to be installed for this to work. Maybe we should have a comment to indicate that.